rest-server/auth-server/src/main/scala/com/actionml/authserver/routes/AuthorizationController.scala

```diff
@@ -3,7 +3,7 @@ package com.actionml.authserver.routes
 import akka.http.scaladsl.model.headers.{BasicHttpCredentials, HttpChallenges}
 import akka.http.scaladsl.server.AuthenticationFailedRejection.{CredentialsMissing, CredentialsRejected}
 import akka.http.scaladsl.server._
-import com.actionml.authserver.model.Client
+import com.actionml.authserver.model.{Client, ClientId}
```

Comment:
The word "client" means too many things. You are talking about an ID for something like a user account, right? Not a client to some server.